.travis.yml

```diff
@@ -24,7 +24,6 @@ install:
   - conda config --add channels omnia
   - conda config --add channels conda-forge
   - conda config --add channels mosdef
-  - conda config --add channels cmelab
```

Comment:
Is all of omnia's work on conda-forge now?

Reply:
I don't think so: https://anaconda.org/search?q=openmm

(OpenMM is our only `omnia` dep, right?)

Reply:
AFAIK, yes just openmm, http://docs.openmm.org/latest/userguide/application.html#installing-openmm currently they haven't migrated all over to conda-forge, so you need omnia and conda-forge